### PR TITLE
Complete overhaul of frontend/cleanup

### LINKED ISSUE #109: Remove all remnants of old code/cleanup
The codebase is not pretty atm, and there's alot of unused code around the place. We should remove all of the js code and dependencies and do some general cleanup.

### PR DESCRIPTION
**Pull request by LudvigHz**
This PR bumps a lot of stuff, removes even more stuff and fixes some stuff :shipit: 

- Remove a bunch of deps, and clean up. The same dependencies were installed with cdns and npm :upside_down_face:.
- Bump AdminLTE to v3 for a more modern look.
- Remove webpack, bacause it's useless. The only thing it did was copy some files from node_modules during build. So it was replaced with a simple script.
- Some minor tweaks to the interface.
- General cleanup of the codebase

Resolves #109 (somewhat)

_Also:_
```
yarn audit v1.22.4
0 vulnerabilities found - Packages audited: 233
Done in 0.79s.
```
:fire: :fire:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,29 +20,22 @@
...
+    "bootstrap": "^4.5.0",
     "bootstrap-chosen": "^1.4.2",
-    "bootstrap-datepicker": "^1.7.1",
-    "bootstrap-table": "^1.11.2",
-    "chosen-js": "^1.8.2",
+    "bootstrap-datepicker": "^1.9.0",
+    "bootstrap-table": "^1.16.0",
+    "chosen-js": "^1.8.7",
     "eonasdan-bootstrap-datetimepicker": "^4.17.47",
     "file-loader": "^3.0.1",
```
Comment (odinuge) [inline package.json:31]: :bomb: :radioactive:

File: README.md [hunk 2]
```diff
@@ -86,6 +86,8 @@ We enforce the style guide [PEP 8] with flake8, and [isort] for import sorting.
 $ isort -rc itdagene               # Fix isort errors
 $ black itdagene                # Fix code formatting
 $ tox -e isort -e flake8 -e yapf   # Verify code style
+
+$ make fixme # Fix all formatting
```
Comment (odinuge) [inline README.md:90]: :heart_eyes:

**Comment by odinuge** [comment]
(also, no more `styl` :heart_eyes:)

File: package.json (hunk 1 above)
Comment (LudvigHz) [reply package.json:31]: Hmm, I seem to have missed some unused stuff here :open_mouth:

**Comment by LudvigHz** [comment]
_complete overhaul_ is probably an overstatement. It's really just an overhaul of the js assets and a bump of adminLTE to v3, among some very minor tweaks (i.e. removed some dead links, added some buttons and options to delete pages and events).

But here are some screens of the "new" look:
company
pages

**Pull request merged**